python/conda/meta.yaml

```diff
@@ -20,12 +19,15 @@ requirements:
   build:
     - python
     - cython >=0.13
+    - nomkl # [not win]
```

Comment:
I changed it now and made `nomkl` a *test* requirement, which is the part that is triggered after a successful build, unless one chooses the `--no-test` option of `conda build`. The runtime dependency is gone.
Next I'm going to investigate the point you make @dmpelt. Conda tags our builds neither with a numpy version nor the nomkl flag, so any runtime version of numpy should be fine. I'll check it just to be sure.